lib/serviceAgents.js

```diff
@@ -59,23 +60,34 @@ export async function generate({serviceType} = {}) {
   // app identity is the controller of the service agent
   const {id, keys: {capabilityInvocationKey}} = getAppIdentity();
   const appIdentitySigner = capabilityInvocationKey.signer();
-
+  logger.info('======DEBUG', {id, capabilityInvocationKey});
```

Comment:
@gannan08 I'm assuming you tried this, but does the bedrock-core logger not have a `logger.debug`?

Reply:
This PR is in DRAFT in order to discuss new changes. The logging statement will be removed.